Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -611,10 +626,40 @@ protected override void RenderContents (HtmlTextWriter writer)
         RenderNodes(writer, _nodes, true, nodeIDAlgorithm, 0);
       }
       writer.RenderEndTag();
+
+      RenderAriaTreeAlertBox(writer);
+      RenderAriaPossibleChildrenDescription(writer);
+
       foreach (DropDownMenu menu in _menuPlaceHolder.Controls)
         menu.RenderAsContextMenu(writer);
     }
 
+    private void RenderAriaTreeAlertBox (HtmlTextWriter writer)
+    {
+      writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassScreenReaderText);
+      writer.AddAttribute(HtmlTextWriterAttribute2.AriaLive, HtmlAriaLiveAttributeValue.Assertive);
```

Comment:
we discussed whether role=alert is better than aria-live. Did it not work or was there another reason that spoke against it?

Reply:
Just seen the comment earlier on this. 

> Aria live seems to work more consistently than role=alert, so I would stick with this, as it essentially does the same thing (we dont care about aria-atomic) https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role
@a-ctor  do you have any additional thoughts on this?

Otherwise, done.

Reply:
Since role=alert basically just sets aria-atomic and aria-live, aria-atomic might even induce the problem the screenreader seems to have